### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dagbdagb**
Some additions to README w.r.t. Microsoft Teams. And gentoo.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +57,7 @@ git clone https://github.com/ammen99/wf-recorder.git && cd wf-recorder
 meson build --prefix=/usr --buildtype=release
 ninja -C build
 ```
-Optionally configure with `-Ddefault_codec='codec'`. The default is libx264. Now you can just run `./build/wf-recorder` or install it with `sudo ninja -C build install`.
+Optionally configure with `-Ddefault_codec='codec'`, or modify meson_options.txt. The default codec is libx264. Now you can just run `./build/wf-recorder` or install it with `sudo ninja -C build install`.
```
Comment (ammen99) [inline README.md:60]: I don't see why modifying `meson_options.txt` is a good option at all.

File: README.md [hunk 2]
```diff
@@ -79,6 +85,20 @@ To set a specific output format, use the `--muxer` option. For example, to outpu
 wf-recorder --muxer=v4l2 --codec=rawvideo --file=/dev/video2
 ```
 
+Microsoft Teams appears to be a bit picky. Install v4l2loopback.ko like this:
```
Comment (ammen99) [inline README.md:88]: Seems like a good write-up, but I do not think it really fits into the README, maybe we can add this to the wiki and add a link from the readme to it?

**Review by ammen99 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (dagbdagb) [reply README.md:60]: Frankly, I had no clue whatsoever what I should put in front of  `-Ddefault_codec=`, so I just fired up vim.....

File: README.md (hunk 2 above)
Comment (dagbdagb) [reply README.md:88]: Absolutely. Can give it a try later, ran out of steam just now.  :-)

**Review by ammen99 — CHANGES_REQUESTED** [review]
A little late, but looking over this, it will be better to put this information in the wiki, in a new page with workarounds for individual applications like MS Teams. Closing the PR, but feel free to extend the wiki.

File: README.md (hunk 1 above)
Comment (eli-schwartz) [reply README.md:60]: BTW you can run `meson configure` and it will print out a description of all build options and what their possible values are. :)

**Pull request closed without merging**